src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://mcr-codes-weather-app.herokuapp.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+  axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        setErrorMessage(
+          "If that was a real place, you would be looking at the forecast right now. Take a brolly anyway just in case"
+        );
+        console.error("Location is not valid", error);
```

Comment:
Since theres a UI element informing the user of the error the console logs here can be removed

Reply:
Removed.